### ISSUE TITLE
Bert Client: ZeroMQ Socket lingering issue after timeout.

### ISSUE THREAD

**Issue by NekoApocalypse** [hanxiao/bert-as-service#187]
**Prerequisites**

> Please fill in by replacing `[ ]` with `[x]`.

* [x] Are you running the latest `bert-as-service`?
* [x] Did you follow [the installation](https://github.com/hanxiao/bert-as-service#install) and [the usage](https://github.com/hanxiao/bert-as-service#usage) instructions in `README.md`?
* [x] Did you check the [FAQ list in `README.md`](https://github.com/hanxiao/bert-as-service#speech_balloon-faq)?
* [x] Did you perform [a cursory search on existing issues](https://github.com/hanxiao/bert-as-service/issues)?

**System information**

- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
- TensorFlow installed from (source or binary):
- TensorFlow version: 1.4.0
- Python version: 3.5.2
- `bert-as-service` version: 1.6.9

---

When I run this script with no active server:

```python
import sys
from bert_serving.client import BertClient

try:
    bert_client = BertClient(timeout=500)
except TimeoutError as e:
    print(e)
sys.exit(0)
```

A time out error is raised, then the script refuses to exit even if sys.exit() is called.

After a brief search on Stack Overflow, I found the following questions:

https://stackoverflow.com/questions/44273941/python-script-not-terminating-after-timeout-in-zmq-recv
https://stackoverflow.com/questions/17140417/termination-of-python-script-while-using-zeromq-with-dead-server

The default behavior of ZMQ is to keep the socket instance alive indefinitely even after timeout. The 'dead' socket instance blocks the script from exiting in the above example. I am not sure if this is going to cause other side-effects in a different setting, but I believe this should be fixed.

Probably we should have LINGER = 0 or some small number (like 200-ish) ?

**Comment by hanxiao** [comment]
thanks for diagnosing and providing details. Will check and fix soon.

**Comment by hanxiao** [comment]
fyi, this issue are fixed in #188 and the new feature is available since 1.7.0. Please do the following for the upgrade:
```bash
pip install -U bert-serving-server bert-serving-client
```

**Comment by NekoApocalypse** [comment]
Thanks. Keep up the good work!
